build/Microsoft.DotNet.Cli.Test.targets

```diff
@@ -42,7 +42,7 @@
           TestProject=%(TestProjects.ProjectPath);
           TestProjectName=%(TestProjects.OutputName);
           TestResultXmlDir=$(TestResultXmlDir);
-          ToolPath=$(Stage0Directory);
+          ToolPath=$(Stage2Directory);
```

Comment:
Can you not make this update right now? It will complicate my PR considerably because of the globing.